lib/chai/utils/inspect.js

```diff
@@ -230,6 +241,28 @@ function formatArray(ctx, value, recurseTimes, visibleKeys, keys) {
   return output;
 }
 
+function formatTypedArray(ctx, value, recurseTimes, visibleKeys, keys) {
+  var output = [];
+
+  // Truncating TypedArray if it has more than 1000 elements
+  var l = value.length > 1000 ? 5 : value.length;
```

Comment:
Yes I thought this was a little bit strange too, but I had no idea how you guys would like the representation of the `Array` to be, are there any guidelines?